src/GlobalScreen/Scope/MetaBar/Factory/NotificationCenter.php

```diff
@@ -21,7 +22,7 @@
  * Class NotificationCenter
  * @author Fabian Schmid <fs@studer-raimann.ch>
  */
-class NotificationCenter extends AbstractBaseItem implements isItem, hasSymbol
+class NotificationCenter extends AbstractBaseItem implements hasSymbol
```

Comment:
Why is this removed?

Reply:
cause this is already required in AbstractBaseItem.
Im aware that this is no argument to remove a requirement but since the NotificationCenter has no other depencdency to item except trhough teh abstract base item this requirement should stay and fall with it.